.github/workflows/nightly.yaml

```diff
@@ -34,14 +34,31 @@ permissions: read-all
 jobs:
   cirq-stable:
     name: Nightly Pytest Cirq stable
-    uses: ./.github/workflows/nightly-pytest.yaml
+    uses: ./.github/workflows/_nightly-cirq-test.yaml
     with:
       args: ''
       reason: '(nightly)'
+      cirq-release: 'stable'
 
   cirq-pre:
     name: Nightly Pytest Cirq pre-release
-    uses: ./.github/workflows/nightly-pytest.yaml
+    uses: ./.github/workflows/_nightly-cirq-test.yaml
     with:
       args: '--pre'
```

Comment:
Note the `--pre` argument applies to everything on the dependency chain and can cause spurious failures if some of cirq dependencies publish a breaking pre-release.

```suggestion
      args: 'cirq-core~=1.0.dev'
```

(I have tested `pip install -U "cirq-core" "cirq-core~=1.0.dev0"` installs the latest Cirq pre-release.)

Reply:
Thanks for that info – I actually didn't know the effect of `--pre` extended to other packages. I made this change in the latest commits.